cli_tools/gce_vm_image_import/importer/api_inflater.go

```diff
@@ -87,8 +95,22 @@ func (inflater *apiInflater) inflate(loggableBuilder *service.SingleImageImportL
 		return persistentDisk{}, inflationInfo{}, err
 	}
 
-	// Cleanup the shadow disk ignoring error
-	defer inflater.computeClient.DeleteDisk(inflater.args.Project, inflater.args.Zone, cd.Name)
+	defer func() {
+		// Cleanup the shadow disk ignoring error
+		inflater.computeClient.DeleteDisk(inflater.args.Project, inflater.args.Zone, cd.Name)
+
+		// Unblock the "cancel" function by receiving the msg.
```

Comment:
Where is the cancel func blocked exactly? The other two selects on inflater.cancelChan are non-blocking due to the presence of default: option.

Reply:
The "cancelChan <- xxx" will block the cancel() function